controller/users.go

```diff
@@ -863,7 +868,7 @@ func loadKeyCloakIdentity(appl application.Application, user account.User) (*acc
 }
 
 // ConvertToAppUser converts a complete Identity object into REST representation
-func ConvertToAppUser(request *goa.RequestData, user *account.User, identity *account.Identity) *app.User {
+func ConvertToAppUser(request *goa.RequestData, user *account.User, identity *account.Identity, isAuthenticated bool) *app.User {
```

Comment:
Add the attribute description to the function comment what isAuthenticated means please.

Reply:
thanks 287005aa5744cef0eeea2c44c3c42b6f462d2d97